src/content/docs/swate/add-building-blocks.mdx

```diff
@@ -4,11 +4,24 @@ lastUpdated: 2024-04-26
 authors: 
   - kevin-frey
   - stella-eggels
+  - patrick-blume
 ---
 
+import { Steps } from '@astrojs/starlight/components';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+<Tabs syncKey="host">
```

Comment:
rename syncKey to more descriptive "swateHost"

Reply:
fixed